website/blog/2022-12-22-block-patterns.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Block Patterns
+description: Intro to block patterns and examples how to use them
+slug: block-patterns
+authors: obradovic
+date: 2023-03-01
+tags: [eightshift, boilerplate, block, patterns]
+hide_table_of_contents: false
+---
+
+Although Block Patterns may be similar to Block Variations, there are some differences between the two. This blog post will cover what Block Patterns are and how to use them.
+<!--truncate-->
+
+## What are Block Patterns and why use them?
+Block Patterns are predefined block layouts which allow you to speed up adding new content and make it look more consistent. The basic idea is to have a ready template which consists of multiple blocks, that already have predefined options, and all you have to do is add content to these blocks.
+
+They can be used in multiple ways. You can create sections composed of few blocks which you can then easily insert anywhere on the site. You can also create complex page layouts which could be used for writing blog posts, for example.
+
+## Technical differences
+As explained in the [Eightshift Development kit documentation](https://eightshift.com/docs/basics/blocks-patterns), the main difference between variations and patterns are:
+- variations provide data using `manifest.json`, while patterns are registered using PHP
+- variations appear in the editor in the same tab as blocks, while patterns appear in their own tab
+
```

Comment:
Maybe also add that variations are presets of just one block, and patterns are just a "named collection of blocks"?

Reply:
I can add it, although I did mention in the section above that patterns are collection of blocks. So I'm not sure if that would be repeating it too soon 😄